### ISSUE TITLE
swallowed exception

### ISSUE THREAD

**Issue by honzasmuk** [wultra/powerauth-restful-integration#69]
https://github.com/wultra/powerauth-restful-integration/blob/fbd36af242e2cd8a6c2e289144c528fb9bb1d1c7/powerauth-restful-security-spring/src/main/java/io/getlime/security/powerauth/rest/api/spring/controller/ActivationController.java#L114

The cause exception is swallowed instead of rethrown or at least logged.

In our case the cause was SunCertPathBuilderException (wrong trust configuration) and we had to debug to find it out :)

**Comment by petrdvorak** [comment]
Yes, this is a valid issue - we need to log the cause exception.
